bbb/schemas/payload.yml

```diff
@@ -0,0 +1,77 @@
+id:          http://payload.bbb/v1.schema
```

Comment:
I'd love to have it published over there! I assume we need to manually update it whenever it changes?